generated/ApiResponse.h

```diff
@@ -0,0 +1,46 @@
+#ifndef ORG_OPENAPITOOLS_MODEL_ApiResponse_H_
+#define ORG_OPENAPITOOLS_MODEL_ApiResponse_H_
+
+#include "serialization.h"
+
+#include <optional>
+
+#include <string>
+
+namespace org {
+namespace openapitools {
+namespace model {
+
+/**
+ * \brief Describes the result of uploading an image resource
+ */
+struct ApiResponse {
+    static const char *dtoName() { return "swagger::dto::ApiResponse"; }
```

Comment:
Is `dtoName` still needed? I copied it from the old template.

Reply:
Its nice to have this for debug printing purposes. But I could imagine that [`typeid(T).name()` might do the trick](https://godbolt.org/z/AuhLCt), too. One could even [demangle](https://stackoverflow.com/questions/281818/unmangling-the-result-of-stdtype-infoname) this, apparently. 

All of that would happen in the user's code (i.e. Bdrive) and wouldn't add complexity to the dto generation, which is nice.

Reply:
Neat. I take it we can remove this entirely, correct? Users can make us of `typeid` and don't need us to provide `dtoName`.